packages/celotool/src/lib/generate_utils.ts

```diff
@@ -122,7 +122,7 @@ export const getAddressesFor = (accountType: AccountType, mnemonic: string, n: n
 export const getStrippedAddressesFor = (accountType: AccountType, mnemonic: string, n: number) =>
   getAddressesFor(accountType, mnemonic, n).map(strip0x)
 
-export const getValidators = (mnemonic: string, n: number) => {
+export const getValidatorsInformation = (mnemonic: string, n: number) => {
```

Comment:
Good suggestion to rename, but it could be more specific.
```suggestion
export const getValidatorKeys = (mnemonic: string, n: number) => {
```